reference/technical_vision/1.5/normalizers_denormalizers.rst

```diff
@@ -0,0 +1,64 @@
+Normalizers & Denormalizers [WIP]
+=================================
+
+The PIM heavily uses the Serializer component of Symfony http://symfony.com/doc/2.7/components/serializer.html.
+
+Especially, classes which implement,
+ - NormalizerInterface to transform object to array
+ - DenormalizerInterface to transform array to object
+
+We have a lot of different formats and for backward compatibility reason, we never re-organized them.
+
+```
+Pim/Bundle/CatalogBundle
```

Comment:
this is completely broken because of rst.
We should use ` namespace : what is it for`.
We should put it in the same chapter that I talked about for architecture